v2-migration/src/main/resources/META-INF/rewrite/aws-sdk-java-v1-to-v2.yml

```diff
@@ -21,16 +21,20 @@ tags:
   - aws
   - sdk
 recipeList:
+  - software.amazon.awssdk.v2migration.AddS3EventNotificationDependency
```

Comment:
We should update tm recipe as well (which includes all of this recipe)

https://github.com/aws/aws-sdk-java-v2/blob/master/v2-migration/src/main/resources/META-INF/rewrite/aws-sdk-java-v1-to-v2-with-tm.yml